BackendTests/UnitTests/PopulationInNorwayDatasetTypeTests.cs

```diff
@@ -0,0 +1,112 @@
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using Backend.Models.Base.JsonStat;
+using Backend.Models.SSBPopulationStatistics.GraphQLTypes;
+using Backend.Models.SSBPopulationStatistics.POCO;
+using GraphQL;
+using GraphQL.Execution;
+using Shouldly;
+using Xunit;
+
+namespace BackendTests.UnitTests
+{
+    public static class PopulationInNorwayDatasetTypeTests
+    {
+        [Fact]
+        public static void PopulationInNorwayDatasetCorrectOutput()
+        {
+            var actor = new PopulationInNorwayDatasetType();
+
+            Assert.NotNull(actor.GetField("value"));
+
+            var expected = new LabeledValue()
```

Comment:
When not using `new()` _and_ using curly brackets to set the properties, the parenthesis are redundant when instantiating an object and can be removed. Change or let stay as is, I don't have any preference.